app/controllers/types_controller.rb

```diff
@@ -0,0 +1,63 @@
+class TypesController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
@basti I need that line because that is the name of the method, where I redirect a user if is not sign in. I checked I do not need that method in attributes because of type/type_id/attribute/attribute_id. Should I change it?

Reply:
@basti I wrote that in a better way using cancan.